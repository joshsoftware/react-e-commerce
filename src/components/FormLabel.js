import React from 'react';
import { Label } from 'reactstrap';
import PropTypes from 'prop-types';

const FormLabel = ({ field, labelText }) => {
  return <Label for={field}>{labelText}</Label>;
};

export default FormLabel;

FormLabel.propTypes = {
  field: PropTypes.string,
  labelText: PropTypes.string
};
