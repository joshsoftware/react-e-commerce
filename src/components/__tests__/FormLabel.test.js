import FormLabel from '../FormLabel';
import { render } from '@testing-library/react';
import React from 'react';

describe('FormLabel Component', () => {
  it('Must return FormLabel Component', () => {
    const { asFragment } = render(<FormLabel />);
    expect(asFragment()).toMatchSnapshot();
  });
});
