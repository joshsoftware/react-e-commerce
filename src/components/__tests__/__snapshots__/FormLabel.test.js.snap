// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormLabel Component Must return FormLabel Component 1`] = `
<DocumentFragment>
  <label
    class=""
  />
</DocumentFragment>
`;
